addon/components/horizontal-bar-chart.js

```diff
@@ -33,13 +34,57 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   // ----------------------------------------------------------------------------
   // Data
   // ----------------------------------------------------------------------------
-
   finishedData: Ember.computed.alias('sortedData'),
 
   // ----------------------------------------------------------------------------
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+   * Overrides values in addon/mixins/axis-titles.js
+   * Location of axis title should track the actual axis
+   *   - If there are both positive and negative values
+   *   - Since the chart axis will be close to center
+   * @override
+   */
+  xAxisPositionX: Ember.computed('graphicWidth',
+      'xTitleHorizontalOffset', function() {
+    var position = this.get('graphicWidth') / 2 ;
+    if (!Ember.isNone(this.get('xTitleHorizontalOffset'))) {
+      position += this.get('xTitleHorizontalOffset');
+    }
+    return position;
+  }),
+
+  /**
+   * X Axis Titles needs extra padding, else will intersect with the lowest bar
+   * @override
+   */
+  xAxisPositionY: Ember.computed('graphicBottom', 'xTitleVerticalOffset', function(){
+    return this.get('graphicBottom') + this.get('xTitleVerticalOffset');
+  }),
+
+  /**
+   * @override
```

Comment:
It was in the comment, but if we don't have xAxisPositionY, the label text will be be overlaid on top of the lowest bar.